GlimpseImputationPipeline/glimpse_docker/build_base_and_extension_docker.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
+# This branch is using the following configuration.
+# To replicate this branch, run this script with the following arguments:
+# build_base_and_extension_docker.sh -r "https://github.com/kachulis/GLIMPSE.git" -b ck_checkpoint_clean -t "glimpse_build_test"
+
+set -Eeuo pipefail
+
+script_dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
```

Comment:
why would `script_dir=$(dirname "${BASH_SOURCE[0]}")` not work?

Reply:
To be completely honest, I used a bash script template for this, but I did some research and according to [this](https://www.ostricher.com/2014/10/the-right-way-to-get-the-directory-of-a-bash-script/), using just `BASH_SOURCE` would break if there are symlinks involved